src/Microsoft.OData.Core/Parameterized.Microsoft.OData.Core.cs

```diff
@@ -5215,9 +5215,12 @@ internal static string UriQueryExpressionParser_CloseParenOrOperatorExpected(obj
         /// <summary>
         /// A string like "Only $filter and $search query options are allowed within $count."
         /// </summary>
-        internal static string UriQueryExpressionParser_IllegalQueryOptioninDollarCount()
+        internal static string UriQueryExpressionParser_IllegalQueryOptioninDollarCount

```

Comment:
These changes were auto-generated by running the `Microsoft.OData.Core.tt` and `Parameterized.Microsot.OData.Core.tt` T4 templates. I think the files might have been previously updated manually, leading to some inconsistencies between the template files and the code files, which explains why they have been modified after re-running the T4 files.